src/services/apps-service.ts

```diff
@@ -77,7 +77,7 @@ export const cloneAppTemplateAndLoadManifest = async (
 };
 
 export const createFeatures = async (ctx: AppCreateCommandTasksContext) => {
-  const defaultVersion = await defaultVersionByAppId(ctx.appId!, false);
+  const defaultVersion = await defaultVersionByAppId(ctx.appId!, undefined, false);
```

Comment:
here is an example of "forced assigning" on the default parameters which is a bad way to solver the problem. (related to my other comment)